canonical-websites/www.ubuntu.com
# Fixed icons disappearing when hovering on Firefox

## Done

- Fixed icons disappearing when hovering on Firefox by changing '#' in the icon background-image CSS to '%23'

## QA

- Run the demo in Firefox and check that the Cloud, Server, Containers etc icons turn orange when hovering over them


## Issue / Card
Fixes #2313
